lib/core/service/feature_manager/feature_manager.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:app/core/domain/event/entities/event_user_role.dart';
+import 'package:app/core/service/feature_manager/feature_visibility_strategy.dart';
+import 'package:app/graphql/backend/schema.graphql.dart';
+
+class FeatureManager {
```

Comment:
this should be specific to event, should be inside service/event
also the name should be EventFeatureManager, FeatureManager only cause misunderstanding with app's features